modules/apps/27-interchain-accounts/controller/types/msgs_test.go

```diff
@@ -93,12 +100,14 @@ func TestMsgRegisterInterchainAccountValidateBasic(t *testing.T) {
 	}
 }
 
-func TestMsgRegisterInterchainAccountGetSigners(t *testing.T) {
+func (suite *TypesTestSuite) TestMsgRegisterInterchainAccountGetSigners() {
```

Comment:
I think there is value in changing all `GetSigners` tests to essentially use this setup. 

If someone changes a protobuf signer msg annotation at some point then they would break/fail.